core/src/main/java/me/mykindos/betterpvp/core/command/commands/admin/CustomGiveCommand.java

```diff
@@ -50,13 +50,22 @@ public void execute(Player player, Client client, String... args) {
 
         int count = 1;
 
+        if (args.length > 2) {
+            try {
+                count = Integer.parseInt(args[2]);
+            } catch (NumberFormatException ignored) {
+
```

Comment:
Send a message to player indicating they need to provide a valid number

Reply:
Implemented